index.js

```diff
@@ -5,6 +5,7 @@ const http = require('http')
 const Ajv = require('ajv')
 const Request = require('./lib/request')
 const Response = require('./lib/response')
+const fixProto = require('./util')
```

Comment:
```suggestion
const freezePrototype = require('./util')
```
I don't think that this name explain what this method does. Maybe freeze can explain more. What do you think?

Reply:
Hah, I used that name at first, don't know why I changed it to `fixProto` 😅 
Anyway renamed, thanks.